redi-connect-front/public/locales/en/translation.json

```diff
@@ -0,0 +1,51 @@
+{
+  "button": {
+    "signup1": "sign-up",
+    "signup2": "sign-up now!",
+    "login": "log-in",
+    "contact": "say hello!"
+  },
+  "home": {
```

Comment:
I think its safe to leave the home out here since all the components used here are pretty specific

Reply:
solved